docs/user-guide/ZE-install-checklist.md

```diff
@@ -1,3 +1,34 @@
 # Zowe Explorer installation checklist
 
-Content goes here. Be patient.
\ No newline at end of file
+This checklist outlines the required steps for a first-time installation of Zowe Explorer.
+
+:::info Required roles: systems administrator, devops architect, security administrator, systems programmer
+:::
+
+The checklist includes a brief description of the steps  required for installation of Zowe Explorer. The checklist also identifies the roles that are typically required to complete the step, which enables the pre-installation planning team to focus on the tasks for which they are responsible.
+
+For a printable version of this checklist, <a href="/stable/Zowe_CLI_Installation_Checklist.xlsx" target="_blank">click here</a>.
+
+## Preparing for installation
+
+| Step        | Description | Role       | Time Estimate |
+| ----------- | ----------- | ---------- | ------------- |
+|[Addressing Zowe Explorer system requirements](../getting-started/ZE-system-reqs.md) | Check the following items: <ul><li>your operating system</li><li>development environments</li></ul> | Systems administrator | 15 min. |
+| (Optional) [Configuring z/OSMF](../user-guide/cli-install-configure-zosmf.md) | Confirm that z/OS components, region sizes, and user IDs meet Zowe Explorer requirements. <br/> Required step when using a z/OSMF profile connection. | Systems programmer | 40 min. |
+| (Optional) [Configuring z/OSMF security](../user-guide/cli-install-configure-zosmf-security.md) | Configure security for: <ul><li>SAF access to REST endpoints</li><li>z/OS console REST interface</li><li>z/OS data set and file REST services</li></ul> Required step when using a z/OSMF profile connection.| Security administrator| 50 min. |
```

Comment:
@adam-wolfe @JillieBeanSim For these two z/OSMF steps, is the note "Required step when using a z/OSMF profile connection." sufficient?

Reply:
Seems good to me.